plugin/testdata/request-validator-simplified.lua

```diff
@@ -0,0 +1,53 @@
+local mime = require "go.mime"
+local parse_mime_type = mime.parse_mime_type
+
+local typedefs = require "kong.db.schema.typedefs"
+
+-- fallback `nkeys` function as `table.nkeys()` LuaJit function is not available for use
+local function nkeys(t)
+  local count = 0
+  for _ in pairs(t) do count = count + 1 end
+  return count
+end
+
+
+local function validate_content_type(entity)
+  if entity == nil or entity == "" then
+    return false, "content type cannot be empty"
+  end
+
+  local t, params = parse_mime_type(entity)
+  if not t then
+    return false, "invalid value: " .. entity
+  end
+
+  if params and nkeys(params) > 1 then
+    -- RFC does not claim the behavior of multiple parameters.
+    -- Hence supports only one parameter for now(normally is 'charset').
+    return false, "does not support multiple parameters: " .. entity
+  end
+
+  return true
+end
+
+return {
+  name = "request-validator-simplified",
```

Comment:
WDYT about changing this filename to `request-validator-mime.lua` and the name to `request-validator-mime` since we are just validating the mime types and not actually the full request-validator.

Reply:
Done! https://github.com/Kong/goks/pull/56/commits/fe359474d68a4e7211751c1064bffcec38aefb7d